command/shell.go

```diff
@@ -118,6 +118,7 @@ func (c *shellExec) Execute(ctx context.Context, _ client.Communicator, logger c
 	env := map[string]string{
 		util.MarkerTaskID:   conf.Task.Id,
 		util.MarkerAgentPID: strconv.Itoa(os.Getpid()),
+		"GOCACHE":           filepath.Join(c.WorkingDir, ".gocache"),
```

Comment:
Check if it isn't already set in this case.

Reply:
it's in the declaration of the map, we don't do anything to populate the map in shell commands.

Reply:
I don't understand. I mean, check if it's already set. Don't set it in the map if it is.